internal/storage/cache/cache_test.go

```diff
@@ -0,0 +1,80 @@
+package cache_test
```

Comment:
```suggestion
package cache
```

if we move this test in the same package when can use the unexported helper methods as requested above.